from src.application.shared.exception.exceptions import ValidationException